### PR TITLE
LPD-40988 Export includes unselected portlets if a widget page template with a web content display portlet is exported

### PR DESCRIPTION
**Pull request by DaniSzimko**
https://liferay.atlassian.net/browse/LPD-40988

reset PORTLET_DATA_ALL after exporting metadata

### CODE REVIEW

**Comment by DaniSzimko** [comment]
ci:test:sf

**Comment by DaniSzimko** [comment]
ci:test:headless-functional

**Comment by DaniSzimko** [comment]
ci:test:headless-staging

File: modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts [hunk 1]
```diff
@@ -45,6 +45,29 @@ export class ExportImportPage {
...
+		await this.fileSelector.click();
+
+		const fileChooser = await fileChooserPromise;
+
+		await fileChooser.setFiles(folderPath);
+
+		await this.continueButton.click();
+
+		await this.page.waitForLoadState('domcontentloaded');
+		await this.page.waitForTimeout(1000);
```
Comment (magjed4289) [inline modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts:65]: is it possible to change the timeout to something else?

File: modules/test/playwright/tests/export-import-web/import.spec.ts [hunk 2]
```diff
@@ -61,6 +71,78 @@ async function getSiteHomePageScreenshot(
...
+			await exportImportPage.downloadExportProcess(exportName);
+
+		await exportImportPage.goToImport();
+
+		await exportImportPage.checkItemInNewlyCreatedImportProcess(
+			exportFilePath,
+			'Wiki'
+		);
+	}
+);
```
Comment (magjed4289) [inline modules/test/playwright/tests/export-import-web/import.spec.ts:144]: How do we handle what is created in the test elements? Maybe it's possible to remove the data using the `apiHelpers.data.push({id: ${element}.id, type: ' ${elementType}'});`?

**Review by magjed4289 — CHANGES_REQUESTED** [review]
I have left some comments for the test

File: modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts (hunk 1 above)
Comment (DaniSzimko) [reply modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts:65]: The number is already a reduced number, but I believe we need that time to upload the file.

File: modules/test/playwright/tests/export-import-web/import.spec.ts (hunk 2 above)
Comment (magjed4289) [reply modules/test/playwright/tests/export-import-web/import.spec.ts:144]: I didn't notice the isolated test is being used here, so there's no need to remove anything

**Comment by DaniSzimko** [comment]
ci:test:sf

**Review by magjed4289 — APPROVED** [review]
looks good test-wise

**Comment by DaniSzimko** [comment]
ci:forward:force

**Comment by DaniSzimko** [comment]
ci:forward:force

**Pull request closed without merging**